src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> fruits = new HashMap<>();

```

Comment:
The `fruits` map should be encapsulated. Consider changing its visibility to private and provide a method to access the data in a controlled manner if needed.